src/components/OptionsListSkeletonView.tsx

```diff
@@ -17,12 +17,14 @@ function getLinedWidth(index: number): string {
 
 type OptionsListSkeletonViewProps = {
     shouldAnimate?: boolean;
+    gradientOpacity?: boolean;
```

Comment:
The name is a bit confusing, it suggests this prop should be a number
Would `gradientOpacityEnabled` be better?

Reply:
Sounds good, renamed